data/dummy-sleep-data.js

```diff
@@ -0,0 +1,167 @@
+const sleepTestData = [
```

Comment:
I really love to see this data, but I wonder if it would better fit topically on a different PR?

Reply:
I don't think that would be a problem :)